### DISCUSSION TITLE
How to reselect Home or Mobile profile from the installation process?

### DISCUSSION THREAD

**Discussion by byrongibson** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1183] [Q&A] [answered]
Hello, I just bought and installed Al Dente Pro.  During installation it asked me to select a usage profile, Home or Mobile.  I selected Home, but want to change it to Mobile.  Is there any way to do that besides reinstalling?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @byrongibson,

Unfortunately, at the moment you need to reinstall.

Best,
Matthias

**Reply by byrongibson** [reply]
Thanks, just tried that, but it remembered my settings and didn't do the install wizard again.  I first uninstalled the Helper application from inside Al Dente settings, then dragged the Al Dente app from the Applications folder into trash.  Seems I missed a step.  How do I do a full uninstall?

**Reply by MatthiasKerbl** [reply]
Hi @byrongibson,

We have an [uninstall guide](https://apphousekitchen.com/how-to-uninstall-aldente-pro/) on our website.

Best,
Matthias
